share/hooks/raft/vip.sh

```diff
@@ -86,9 +86,21 @@ function virtualip() {
 # main
 #
 
+if [ -z "${ONE_LOCATION}" ]; then
+    LOCK_LOCATION=/var/lock/one
+else
+    LOCK_LOCATION=$ONE_LOCATION/var/lock
+fi
+
+LOCK_FILE="$LOCK_LOCATION/vip_sh"
+
 ACTION="$1"
 shift
 
+# Start of critical section (opens LOCK_FILE with fd 56)
+exec 56>$LOCK_FILE
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Double quote to prevent globbing and word splitting.